doc/api/colors_api.rst

```diff
@@ -71,3 +71,23 @@ Functions
    same_color
    get_named_colors_mapping
    make_norm_from_scale
+
+Exported colors
+---------------
+
+.. note::
```

Comment:
I don't think this needs to be in a note, can can just be normal prose.

Maybe something like

>  The data used to populate the :ref:`named colors` are exposed as dictionaries that map color name to hex string

?

Reply:
Got it!